### PR TITLE
Downgrade required xtext version

### PR DESCRIPTION
**Pull request by kthoms**
1) Lower Requirement to Xtext 2.14
2) Increment project version to 0.3.0-SNAPSHOT

### CODE REVIEW

File: features/com.itemis.xtext.testing.feature/feature.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <feature
       id="com.itemis.xtext.testing.feature"
       label="Xtext Testing Framework"
-      version="0.2.0.qualifier">
+      version="0.3.0.qualifier">
```
Comment (cdietrich) [inline features/com.itemis.xtext.testing.feature/feature.xml:5]: is there a specific reason to bump version?

Comment (kthoms) [reply features/com.itemis.xtext.testing.feature/feature.xml:5]: My idea is here: Somewhen a version 0.2.0 has been around, and this is now a change to what was available then. Semantic versioning FTW.

Comment (kthoms) [reply features/com.itemis.xtext.testing.feature/feature.xml:5]: Like you prefer... I could roll it back if you want.

Comment (cdietrich) [reply features/com.itemis.xtext.testing.feature/feature.xml:5]: we use the 0.2.0 all the time. so never did semantic version.
i also assume that this enhances comp. not lowers it

**Review by cdietrich — APPROVED** [review]

**Pull request merged**